crates/turborepo-auth/src/lib.rs

```diff
@@ -200,6 +200,12 @@ impl Token {
                 turborepo_api_client::Error::UnknownStatus { code, .. } if code == "forbidden" => {
```

Comment:
I'm unsure if this was ever valid, but keeping in case there's a way this can get hit that I'm not seeing. Timeboxing this fix so not trying to fully trace responses we can get from this endpoint.